requirements.txt

```diff
@@ -5,13 +5,13 @@ gradio==3.24.1
 markdown
 numpy
 Pillow>=9.5.0
-peft==0.2.0
 requests
 rwkv==0.7.3
 safetensors==0.3.0
 sentencepiece
 pyyaml
 tqdm
+git+https://github.com/huggingface/peft
```

Comment:
Just to be sure, is the dev version of peft required? The code seems to run without errors with `peft==0.2.0`.

Reply:
peft 0.2.0 was released March 9th <https://github.com/huggingface/peft/releases> and multi-adapter support was merged April 6th <https://github.com/huggingface/peft/pull/263> so, yes, it's needed? I'm not sure what could lead it to seemingly work on 0.2.0 for you, possibly you accidentally had a different version installed while testing because you were recently testing <https://github.com/johnsmith0031/alpaca_lora_4bit/issues/13> as well? idk.

Reply:
It's definitely needed then.